anyway/localization.py

```diff
@@ -199,7 +199,10 @@
     "RSA_LICENSE_PLATE": "סוג לוחית רישוי",
 }
 
-_cities = pd.read_csv("static/data/cities.csv", encoding="utf-8", index_col=field_names.sign)
+try:
+    _cities = pd.read_csv("static/data/cities.csv", encoding="utf-8", index_col=field_names.sign)
+except FileNotFoundError:
+    pass
```

Comment:
I think it's better to use a function decorated with `lru_cache`

Reply:
to be handled in #1876